src/sagemaker/automl/README.rst

```diff
@@ -0,0 +1,64 @@
+SageMaker Autopilot
+===================
+
+Amazon SageMaker Autopilot is an automated machine learning (commonly referred to as AutoML) solution for tabular
+datasets. It automatically trains and tunes the best machine learning models for classification or regression based
+on your data, and host a serials of models on an Inference Pipeline.
+
+SageMaker AutoML Class
+~~~~~~~~~~~~~~~~~~~~~~
+
+The SageMaker ``AutoML`` class is similar to a SageMaker ``Estimator`` where you define the attributes of an AutoML
+job and feed input data to start the job.
+
+A simple use case of an ``AutoML`` object would be:
```

Comment:
"Here's a simple example of using the `AutoML` object:"

this isn't really a use case as much as it is just usage